rules/do-you-display-information-consistently/rule.md

```diff
@@ -1,27 +1,25 @@
 ---
 type: rule
-archivedreason: 
 title: Do you display information consistently?
-guid: 5b878864-7ced-4b4c-a979-adadf38c7363
 uri: do-you-display-information-consistently
-created: 2015-02-16T01:50:14.0000000Z
 authors: []
-related: []
+related:
+  - descriptive-links
 redirects: []
-
+created: 2015-02-16T01:50:14.000Z
+archivedreason: null
+guid: 5b878864-7ced-4b4c-a979-adadf38c7363
 ---
-
-When redirecting users to another page make sure that the       text is consistent between the two pages.
+When redirecting users to another page, make sure that the text is consistent between the two pages.
 
 <!--endintro-->
 
+![Figure: Good Example - This text is taken from where the "More on SSW Chinafy" link goes](homepage-with-link-to-chinafy.png)
 
-::: good  
-![Figure: Good example - Click "More" and the next page begins with the same information](../../assets/ConsistentInformation.gif)  
-:::
+![Figure: Good Example - You can see the text again here, at the link's destination](chinafy-page.png)
 
-As an example, the SSW website follows a consistent order: a summary of the product/service and its benefit followed by a longer paragraph with more information. When we want to redirect users to a page we simply use the summary part of that page as narrative on the other page (e.g. events list). As a result, user will have a nice flow between pages when navigating around the site.     [See a live example](http://www.ssw.com.au/ssw/Events/).
+As an example, the [SSW website](https://ssw.com.au/) follows a consistent order: a summary of the product/service and its benefit, followed by more information o that topic. 
```

Comment:
@uly1 

- [ ] Please fix typo

Change from
   followed by more information o that topic.
to
    followed by more information **_on_** that topic.